src/main/java/sg/ncl/testbed_interface/StatefulExperiment.java

```diff
@@ -0,0 +1,50 @@
+package sg.ncl.testbed_interface;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by dcszwang on 11/14/2017.
+ */
+@Getter
+@Setter
+public class StatefulExperiment {
+    private String teamId;
+    private String teamName;
+    private Long id;
+    private String name;
+    private String userId;
+    private String description;
+    private ZonedDateTime createdDate;
+    private ZonedDateTime lastModifiedDate;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "lastModifiedDate" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)